packages/analysis_options.yaml

```diff
@@ -10,7 +17,7 @@ linter:
     - avoid_catching_errors
```

Comment:
Can we migrate from handpicked rules to the `lints` package?

Reply:
I can follow up with this and some other analysis related cleanup afterwards.